ee/backup/run.go

```diff
@@ -42,6 +43,7 @@ import (
 )
 
 // Restore is the sub-command used to restore a backup.
+
```

Comment:
This is an unnecessary change.

Reply:
@akon-dey the var Restore x.SubCommand on line 47 is used in other part of the code. So I had to re-introduce to fix the errors

Reply:
looks like github isnt showing the line 47 addition unless you click view changes, not sure if you missed that